lib/commons/builder/boundary_data.rb

```diff
@@ -142,14 +153,15 @@ def ms_fb_to_wikidata
   end
 
   def boundaries_dir
-    @boundaries_dir ||= Pathname.new('boundaries')
+    @boundaries_dir ||= Pathname.new(@boundaries_dir_path)
   end
 
   def index_json_pathname
-    @index_json_pathname ||= boundaries_dir.join('index.json')
+    @index_json_pathname ||= boundaries_dir.join(@index_file)
```

Comment:
It's a tiny thing, but the convention for Ruby in EP projects has been to always use `attr_reader` for instance variables set up in the intializer, to make it easier to change the implementation in the future.